src/test/java/com/puppycrawl/tools/checkstyle/xpath/XpathFileGeneratorAuditListenerTest.java

```diff
@@ -0,0 +1,216 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2018 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.puppycrawl.tools.checkstyle.xpath;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.fail;
+
+import java.lang.reflect.Method;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.puppycrawl.tools.checkstyle.TreeWalkerAuditEvent;
+import com.puppycrawl.tools.checkstyle.api.AuditEvent;
+import com.puppycrawl.tools.checkstyle.api.LocalizedMessage;
+import com.puppycrawl.tools.checkstyle.checks.blocks.LeftCurlyCheck;
+import com.puppycrawl.tools.checkstyle.checks.blocks.RightCurlyCheck;
+
+public class XpathFileGeneratorAuditListenerTest {
```

Comment:
in build output I noticed:
```
[INFO] Running com.puppycrawl.tools.checkstyle.xpath.ElementNodeTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.005 s - in com.puppycrawl.tools.checkstyle.xpath.ElementNodeTest
[INFO] Running com.puppycrawl.tools.checkstyle.xpath.XpathFileGeneratorAuditListenerTest
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suppressions PUBLIC
    "-//Puppy Crawl//DTD Suppressions Xpath Experimental 1.2//EN"
    "http://checkstyle.sourceforge.net/dtds/suppressions_1_2_xpath_experimental.dtd">
<suppressions>
</suppressions>
```

it is not ok to allow print smth to build, build output should have only build related output.
ALL UTs should be silent, and do output only when problem is happened.

Reply:
done